pkg/event_processor/http2_request.go

```diff
@@ -40,9 +39,9 @@ type HTTP2Request struct {
 }
 
 func (h2r *HTTP2Request) detect(payload []byte) error {
-	data := string(payload[0:H2MagicLen])
-	if data != H2Magic {
-		return errors.New("Not match http2 magic")
+	data := string(payload[0:ClientPrefaceLen])
```

Comment:
We need to determine whether the length of the payload is greater than `ClientPrefaceLen`?

Reply:
It hard to meet this situation, but it is better to add it. I will patch it.